tests/AssetTest.php

```diff
@@ -35,8 +35,6 @@ public function registerDataProvider(): array
      */
     public function testAssetRegister(string $type, string $asset, ?string $depend = null): void
```

Comment:
This test wasn't testing anything useful (it was testing that asset manager works and that's already tested) so I've removed it from master.

What is worth testing is the fact that widgets, when used, are registering expected assets.